src/main/java/org/spongepowered/api/plugin/PluginContainer.java

```diff
@@ -114,15 +114,6 @@ default String getName() {
     }
 
     /**
-     * Gets the directory that contains this {@link Plugin}'s assets.
```

Comment:
Well yes, but since it is always `assets/<plugin_id>` I don't think this method is still needed.